### ISSUE TITLE
throwing whenever the statusCode != 200 causes issues

### ISSUE THREAD

**Issue by keyle** [jaggedsoft/node-binance-api#57]
There is no way (AFAIK) to embed your API in Promises, when you simply throw the response on L.89.

_try catch_, or _Promise_ rejection won't get it, and it will just throw and fail.

Recommend you remove L.89 or use Promises instead of a callback.

`if ( response && response.statusCode !== 200 ) throw response;`

**Comment by keyle** [comment]
Example 

```
try {
    binance.marketSell(instrument, balances[coin].available, function (response) {
        resolve(response);
    });
} catch (error) {
    reject(error);
}
```

is the statusCode is different than 200, e.g. Insufficient funds or Lot size, it will just spew it, without catching it in the try/catch, or being able to reject or resolve the promise.

**Comment by jaggedsoft** [comment]
Yes thank you for the suggestion, I agree and this will be fixed in `v0.4.0`
The only reason it has not been fixed is because it's a breaking change

**Comment by keyle** [comment]
Thanks for the quick response. All the best 👍

**Comment by jaggedsoft** [comment]
New release available: `v0.4.0` https://github.com/jaggedsoft/node-binance-api/commit/b5d81071677227ff33041ae6b5e0ed42622f351a
Thanks for your support!
